### DISCUSSION TITLE
Feature Request: LaTex display

### DISCUSSION THREAD

**Discussion by simo-corbo** [pieces-app/support#148] [Ideas]
Hello team,
Firstly I would like to thank you for your incredible work, it has been extremely helpful while preparing for my exams :)
A feature I noticed it would be a great advantage is the possibility to display LaTeX formulas, as I use the often in my notes, an example is shown here:
<img width="416" alt="image" src="https://github.com/pieces-app/support/assets/126069316/68527041-fd42-410e-862c-8e3faee89c05">
Thank you!

**Comment by caleb-at-pieces** [comment]
I like this idea @simo-corbo . I'm thinking this might not be all that difficult given there is likely some `LaTeX -> HTML` package out there

**Comment by tjwhang** [comment]
Yeah, maybe the dev team can utilize KaTeX or something to render equations quickly..

**Reply by mark-at-pieces** [reply]
could you provide some queries that we can use in the copilot module to generate example equations, as well as the model that is being used(will ensure this works across the board, but want to ensure this is solid for the original model being used)

This will help us develop this feat to ensure it is hitting the nail on the head:)

**Reply by igor-nikitin-cc** [reply]
Used model GPT-4o
**Promt:**
`The zeta function is known to be expressed as a sum, and there is also a product representation. How is the product derived from the sum?`
**Answer**
```
The Riemann zeta function \(\zeta(s)\) is a complex function that can be expressed both as an infinite series and as an infinite product. The series representation is given by:

\[
\zeta(s) = \sum_{n=1}^{\infty} \frac{1}{n^s}
\]

for \(\text{Re}(s) > 1\).

The product representation, known as the Euler product, is given by:

\[
\zeta(s) = \prod_{p \, \text{prime}} \frac{1}{1 - p^{-s}}
\]

for \(\text{Re}(s) > 1\).

The Euler product formula is derived from the fundamental theorem of arithmetic, which states that every integer greater than 1 can be uniquely factored into prime numbers. Here's a brief outline of how the product representation is derived from the sum:

1. **Start with the series representation**:
   \[
   \zeta(s) = \sum_{n=1}^{\infty} \frac{1}{n^s}
   \]

2. **Consider the geometric series for each prime \(p\)**:
   \[
   \frac{1}{1 - p^{-s}} = 1 + \frac{1}{p^s} + \frac{1}{p^{2s}} + \frac{1}{p^{3s}} + \cdots
   \]

3. **Take the product over all primes**:
   \[
   \prod_{p \, \text{prime}} \left(1 + \frac{1}{p^s} + \frac{1}{p^{2s}} + \cdots \right)
   \]

4. **Expand the product**: When you expand this product, you get terms of the form \(\frac{1}{n^s}\) where \(n\) is a product of primes, each raised to some power. By the fundamental theorem of arithmetic, every integer \(n\) appears exactly once in this expansion.

5. **Conclude the equivalence**: The expanded product matches the series \(\sum_{n=1}^{\infty} \frac{1}{n^s}\), thus proving the equivalence of the series and product representations for \(\zeta(s)\).

This derivation shows the deep connection between the distribution of prime numbers and the properties of the zeta function.
```
